### ISSUE TITLE
Change dev env to PlatformIO?

### ISSUE THREAD

**Issue by piodabro** [MartinLaza/OpenPRO58#5]

**Comment by piodabro** [comment]
I can make a change with description of setup of dev env.

**Comment by MartinLaza** [comment]
You can try it - I dont care. I am not a fan of Arduino IDE either, nor a good programmer :)

**Comment by MartinLaza** [comment]
Yes, please update the description. That would be great even for me to try something new ;)

**Comment by piodabro** [comment]
There will be a little change in project structure too, cause PIO has different lib/src structure but it's better.
When i finish i will make a pull request ;)

**Comment by piodabro** [comment]
Project moved to Platformio, there is lots of warnings on compilation but thats the result of platformio verbosity ;)
Had to change some code (like Wire.h to HardWire.h cause there is difference in libs).
Moved Adafruit_GFX_AS as local lib. etc.
Compiled code is the same size as in Arduino env.

I'll commit it to my fork of repo. Can You check if it works? Cause i do not have PRO58 yet, its on transit ;)

Fast desc of setup of env:
1. Install Visual Studio Code
2. In VS Code go to Extensions and install Platformio, and restart VS Code
3. In VS Code Open project folder with File->Open Folder...
4. In terminal write "pio run"
it should download required libs, toolchain etc and next build code.

Try it ;)

**Comment by piodabro** [comment]
When You test changes from my fork, we can merge the code.
